test/tools/base64_test.cpp

```diff
@@ -18,69 +18,146 @@
  */
 
 #include "base64.h"
+#include <gtest/gtest.h>
+#include <array>
 
-#include <cgreen/cgreen.h>
+class base64EncodeDecodeTests: public ::testing::Test { 
+protected:
+
+    static constexpr char inputData[] =
+        "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghi"
```

Comment:
Can this be in a single line ?

Reply:
Yes, it is part of multi line/strings testing.